File: JavaScript/EmailSwap.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

function validateAndLowercaseEmail(email) {
  // Regular expression for email validation
  const emailPattern = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;

  if (email.match(emailPattern)) {
    // Convert the email to lowercase
    email = email.toLowerCase();
    console.log(`Valid Email (Lowercase): ${email}`);
  } else {
    console.log("Invalid Email");
  }
}

rl.question("Enter an email address: ", function(email) {
  validateAndLowercaseEmail(email);
  rl.close();
});

File: Python/EmailSwap.py
import re

def validate_and_lowercase_email():
    email = input("Enter an email address: ")

    # Regular expression for email validation
    email_pattern = r"^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$"
    
    if re.match(email_pattern, email):
        # Convert the email to lowercase
        email = email.lower()
        print("Valid Email (Lowercase):", email)
    else:
        print("Invalid Email")

validate_and_lowercase_email()
